src/vs/editor/common/config/editorOptions.ts

```diff
@@ -5404,6 +5478,9 @@ export const enum EditorOption {
 	linkedEditing,
 	links,
 	matchBrackets,
+	// --- Start Positron ---
```

Comment:
Should this have been added at the end of the options to avoid the ordinal reassignment to the enums that followed?

Reply:
On discussion, it seems these are generated during the build (despite being a .ts file) and change upstream all the time. So apparently you cannot rely on the ordinal value and this doesn't matter. Resolving.